compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -45,12 +45,6 @@ static llvm::cl::opt<int> clInlineConstantByteLength(
                    "into a dispatch region or 0 to disable inlining."),
     llvm::cl::init(256));
 
-// TODO(#18457, #18447): Remove once backends support gather fusion.
```

Comment:
https://github.com/iree-org/iree/issues/18457 is still open, can you help review the issue since you're removing the flag? E.g., close the issue if it is outdated. (please also add the context when you close the issue.)